style.css

```diff
@@ -100,3 +100,315 @@ header {
   text-decoration: none;
   list-style: none;
 }
+
+/*
+Works
+*/
+
+.works {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 5px;
+  background-color: white;
+  width: 85%;
+  height: auto;
+  margin: 114px auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-works {
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  flex-direction: column;
+  margin: 20px auto;
+}
+
+.snapshoot-img {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+}
+
+.leftblock {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 5px;
+  margin: 5px auto;
+  width: 295px;
+  overflow: hidden;
+}
+
+.ref {
+  display: flex;
+  flex-direction: row;
+}
+
+.card-works-title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  font-family: 'Poppins', serif;
+}
+
+.client {
+  font-style: normal;
+  font-weight: 400;
+  text-transform: uppercase;
+  color: #344563;
+}
+
+span {
+  margin-right: 12px;
+  text-decoration: none;
+  font-size: 0.8em;
+  font-family: 'Poppins', sans-serif;
+}
+
+.fa-circle {
+  font-size: 60px;
+  color: #c1c7d0;
+  text-decoration: none;
+}
+
+.card-primary-text {
+  width: 295px;
+  height: 76px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+}
+
+.tags {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+}
```

Comment:
- [ ] As per the figma design, tags should be left aligned not centered. Also, there is more space between the tags Kindly fix this.

- Figma Design:
- ![image](https://user-images.githubusercontent.com/60083437/227654177-ae5d2c9c-b37b-429a-a253-6892400200c3.png)

- Your Design:
- ![image](https://user-images.githubusercontent.com/60083437/227654401-f2b088aa-075d-4171-b638-fb077a6ed8de.png)